Makefile

```diff
@@ -144,7 +144,7 @@ test/reference: test/common
 # We want tools/node-modules to run every time package-lock.json is requested
 # See https://www.gnu.org/software/make/manual/html_node/Force-Targets.html
 FORCE:
-package-lock.json: FORCE
+package-lock.json: FORCE tools/node-modules
```

Comment:
This `FORCE` stuff is sufficiently magical that I wondered if this would still work, but the short answer is: it does.

I created an artificial rule trivially dependent on `package-lock.json` to test things, and in every case it performs correctly (and every `make` invocation runs the hook correctly).